src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/SdkDirectoryWorkloadManifestProvider.cs

```diff
@@ -121,11 +121,12 @@ public void RefreshWorkloadManifests()
             _workloadSet = null;
             _manifestsFromInstallState = null;
             _installStateFilePath = null;
-            _workloadSetDominatesInstallState = false;
+            _useManifestsFromInstallState = false;
```

Comment:
The value of `_useManifestsFromInstallState` should be inverted from what `_workloadSetDominatesInstallState` was.  So it should default to true and be set to false if the workload version was specified in the constructor or found in global.json.

Reply:
🤦 I'm stupid sometimes